src/flownet/ahm/_run_ahm.py

```diff
@@ -58,24 +59,32 @@ def _from_regions_to_flow_tubes(
         if tube_regions != []:
             df_regions.append(mode(tube_regions).mode.tolist()[0])
         else:
-            tube_midpoint = network.get_connection_midpoints(int(i))
-            dist_to_cell = []
-            for k in range(1, field_data.grid.get_num_active()):
-                cell_midpoint = field_data.grid.get_xyz(active_index=k)
-                dist_to_cell.append(
-                    np.sqrt(
-                        np.square(tube_midpoint[0] - cell_midpoint[0])
-                        + np.square(tube_midpoint[1] - cell_midpoint[1])
-                        + np.square(tube_midpoint[2] - cell_midpoint[2])
-                    )
+            df_regions.append(None)
+            tube_outside.append(i)
+
+    if tube_outside != []:
+        tube_midpoints = network.get_connection_midpoints()
+        candidate_tubes = set(network.grid.model.unique()) - set(tube_outside)
```

Comment:
Could you update the function description to describe what this part of the function is doing. It's not really clear to me in one go.

Reply:
Now with updated fucntion description (which should have been done when the code was changed in the first place)